code/modules/projectiles/projectile.dm

```diff
@@ -568,9 +568,15 @@
 	return list(angle, p_x, p_y)
 
 /obj/item/projectile/Crossed(atom/movable/AM) //A mob moving on a tile with a projectile is hit by it.
-	..()
-	if(isliving(AM) && (AM.density || AM == original) && !(src.pass_flags & PASSMOB))
-		Bump(AM)
+	. = ..()
+	if(isliving(AM) && !(pass_flags & PASSMOB))
+		var/mob/living/L = AM
+		if(AM == original)		//specific aiming for
+			Bump(AM)
+		if(AM.density)			//blocks projectiles

```

Comment:
Uh, kev, this makes bullets pass through walls and other solid objects. Move this and the previous `if()` check right before it to somewhere before the `if(isliving())` check

Reply:
What? Previous code already checked for living before trying to force bump, you sure?